agentops/langchain_callback_handler.py

```diff
@@ -80,7 +80,7 @@ def on_llm_error(
         llm_event: LLMEvent = self.events.llm[str(run_id)]
         self.ao_client.record(llm_event)
 
-        error_event = ErrorEvent(trigger_event=llm_event, details=str(error), timestamp=get_ISO_time())
+        error_event = ErrorEvent(trigger_event=llm_event, error_type=type(error).__name__, details=str(error), timestamp=get_ISO_time())
```

Comment:
I was thinking more like `error_event = ErrorEvent(trigger_event=llm_event, error=error)` and then the ErrorEvent constructor does the rest:

```
def __init__(self, trigger_event, error):
  self.error_type = error_type=type(error).__name__
  self.details = str(error)
  self.timestamp = get_ISO_time()
```

it's less work for the developer to have to do. more magic

Reply:
or actually maybe like this

```
def __init__(self, trigger_event, error, error_type = None, details = None, timestamp = None):
  self.error_type = error_type or error_type=type(error).__name__
  self.details =  details or str(error)
  self.timestamp = timestamp or get_ISO_time()
```